tests/testing_common.py

```diff
@@ -287,7 +287,10 @@ def _test_save_pretrained(self, model_id, config_cls, config_kwargs):
         model = model.to(self.torch_device)
 
         with tempfile.TemporaryDirectory() as tmp_dirname:
-            model.save_pretrained(tmp_dirname)
+            if safe_serialization:
```

Comment:
I prefer to not pass `safe_serialization=safe_serialization` to test the native behaviour, let me know if this makes sense

Reply:
yes sorry I meant the default behaviour